### ISSUE TITLE
[Catalog] Catalog tiles are missing VoiceOver attributes

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#3162]
b/74073542

## Overview
Catalog example cells that, when tapped, will open an example View Controller should provide accessibility traits as "Button" and a Hint with something like, "Opens the example".

## Reproduction steps
1. Open the Catalog
2. Turn on VoiceOver
3. Activate a component tile
4. Select a Demo cell and listen to the VoiceOver prompt

**Expected result:** VoiceOver includes "Button"
**Actual Result:** VoiceOver reads only "Demo" and the name of the VC

---
#### Internal data
- Associated internal bug: [b/117179015](http://b/117179015)
